tasks/json/parser.js

```diff
@@ -13,4 +13,33 @@
  * where 077b1523-56f0-492a-b954-1269e3acc191 - is the value of the name attribute from the ./test.json file
  * Please NOTE, that you should omit the .html extension
  * Please do the task in the scope of the file. Create a pull request a share it with your mentor.
- */
\ No newline at end of file
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const jsonFile = require('./test.json');
+// console.log(jsonFile.list.entries);
+const mas = jsonFile.list.entries;
+// console.log(mas);
+let parsed = [];
+for (let i = 0; i < mas.length; i++) {
```

Comment:
that's too superfluous, you can simply use .map() method to avoid the cycle and pushing items into the array.

Reply:
please review 'fix_parser' commit 
it contains 'map()' method, but without 'replace()' (